contracts/axelar-gateway/src/auth.rs

```diff
@@ -72,6 +73,10 @@ pub fn validate_proof(
         ContractError::InvalidSignatures
     );
 
+    env.storage()
```

Comment:
consider to encapsulate the extend_ttl logic within a method, making it reusable and clear.
```suggestion
pub fn extend_stoage_instance_ttl(env: &Env) {
    env.storage()
        .instance()
        .extend_ttl(INSTANCE_TTL_THRESHOLD, INSTANCE_TTL_EXTEND_TO);
}
```